tests/test_search.py

```diff
@@ -199,7 +199,44 @@ def GA_GraphColoringInts(edges, fitness):
     population = init_population(8, [0, 1], 4)
 
     return genetic_algorithm(population, fitness)
-
+def test_simpleProblemSolvingAgent():
+    class vacuumAgent(SimpleProblemSolvingAgentProgram):
+        def update_state(self, state, percept):
+            state = percept
+            return state
+        def formulate_goal(self, state):
```

Comment:
You should be adding a space to separate functions. You need to do this for all functions in this test.